packages/mini-browser/src/browser/mini-browser-content.ts

```diff
@@ -297,6 +297,11 @@ export class MiniBrowserContent extends BaseWidget {
 
         const loadIndicator = document.createElement('div');
         loadIndicator.classList.add(MiniBrowserContentStyle.PRE_LOAD);
+        loadIndicator.innerHTML = `<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="100px" height="100px" viewBox="0 0 100 100">
+  <circle cx="50" cy="50" fill="none" stroke="var(--theia-foreground, #949494)" stroke-width="10" r="35" stroke-dasharray="160 59.91">
+    <animateTransform attributeName="transform" type="rotate" repeatCount="indefinite" dur="1s" values="0 50 50;360 50 50" keyTimes="0;1"></animateTransform>
+  </circle>
+</svg>`;
```

Comment:
I don't like that we have to duplicate the code for the spinner. In fact, we should be able to use `codicons` for this spinner:
- https://code.visualstudio.com/api/references/icons-in-labels#animation

Reply:
Hmm, codicon is a good idea. Will try

Reply:
This syntax can only be used in labels, but we can use other icons easily.

Reply:
Did it! Which one do you like more, `ease` or `linear`? `ease` looks better for me.
![Peek 2020-04-08 20-48](https://user-images.githubusercontent.com/19504461/78816434-64d91080-79da-11ea-8471-ddacc5c93f05.gif)
![Peek 2020-04-08 20-47](https://user-images.githubusercontent.com/19504461/78816452-6c001e80-79da-11ea-9bf8-ee647187ed5a.gif)